# isInternalError

Check if `value` is an `InternalError` object

## Description

Currently not supported

## Arguments

- `value`: The value to check

## Returns

Returns `true` if value is an `InternalError`, else `false`

## Since

0.1.0
